xtc/src/stats.rs

```diff
@@ -19,8 +33,9 @@ pub struct StatsData {
 
 impl Default for StatsData {
```

Comment:
hmmm, wouldn't the following code do the same?
```rust
#[derive(Default)]
struct StatsData{...}
```

Reply:
Yes! Thank you